### PR TITLE
Add minimum oracle fee

### PR DESCRIPTION
**Pull request by shargon**
Related to https://github.com/neo-project/neo-devpack-dotnet/pull/370/files#r493376175

### CODE REVIEW

**Review by Tommo-L — DISMISSED** [review]

File: src/Neo.SmartContract.Framework/Services/Neo/Oracle.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@ namespace Neo.SmartContract.Framework.Services.Neo
     [Contract("0x3c05b488bf4cf699d0631bf80190896ebbf38c3b")]
     public class Oracle
     {
+        public const uint MinimumResponseFee = 0_60000000; // OracleRequestPrice=0_50000000 gasForResponse=0_10000000
```
Comment (Tommo-L) [inline src/Neo.SmartContract.Framework/Services/Neo/Oracle.cs:8]: 0_10000000 ?

Comment (Tommo-L) [reply src/Neo.SmartContract.Framework/Services/Neo/Oracle.cs:8]: We don't need to add `OracleRequestPrice`, as it's already part of system fee.
https://github.com/neo-project/neo/blob/ab7ec04925c1e10c9338e3deb8880e93b75ff306/src/neo/SmartContract/Native/Oracle/OracleContract.cs#L126-L127

Comment (shargon) [reply src/Neo.SmartContract.Framework/Services/Neo/Oracle.cs]: Done

**Review by Tommo-L — APPROVED** [review]

**Comment by shargon** [comment]
Ready to merge?

**Pull request merged**